stackslib/src/chainstate/stacks/miner.rs

```diff
@@ -2795,9 +2843,19 @@ impl BlockBuilder for StacksBlockBuilder {
                                         100 - TX_BLOCK_LIMIT_PROPORTION_HEURISTIC,
                                         &total_budget
                                     );
+                                    let mut measured_cost = cost_after;
+                                    let measured_cost = if measured_cost.sub(&cost_before).is_ok() {
+                                        Some(measured_cost)
+                                    } else {
+                                        warn!(
+                                        "Failed to compute measured cost of a too big transaction"
```

Comment:
Does this pass cargo fmt?

Reply:
Yeah, cargo fmt doesn't really always know what to do with key-value macros.